packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputAccount.tsx

```diff
@@ -53,37 +63,37 @@ const CoinmarketFormInputAccount = () => {
             </CoinmarketFormInputLabel>
             <Controller
                 name="cryptoSelect"
-                defaultValue={cryptoList[0]}
                 control={control}
                 render={({ field: { onChange, value } }) => (
                     <Select
                         value={value}
-                        options={cryptoList}
-                        onChange={(selected: any) => {
-                            console.log(selected, value, control);
+                        options={options}
+                        onChange={(selected: CoinmarketCryptoListProps) => {
                             onChange(selected);
                         }}
-                        formatOptionLabel={(option: (typeof cryptoList)[number]) => (
+                        formatOptionLabel={(
+                            option: CoinmarketOptionsGroupProps['options'][number],
+                        ) => (
                             <CoinmarketFormOption>
-                                <CoinmarketFormOptionIcon>
-                                    {account.symbol.toUpperCase() === option.value ? (
-                                        <CoinLogo size={18} symbol={account.symbol} />
-                                    ) : (
-                                        <CoinmarketFormOptionTokenLogo symbol={option.value} />
-                                    )}
-                                </CoinmarketFormOptionIcon>
+                                <CoinmarketFormOptionIcon symbol={option.label} />
                                 <CoinmarketFormOptionLabel>
                                     {shouldSendInSats ? 'sats' : option.label}
                                 </CoinmarketFormOptionLabel>
                                 <CoinmarketFormOptionLabelLong>
                                     {option.cryptoName}
                                 </CoinmarketFormOptionLabelLong>
+                                {option.value &&
+                                    isCryptoSymbolToken(option.value) &&
+                                    cryptoToNetworkSymbol(option.value) && (
+                                        <CoinmarketFormOptionNetwork $elevation={elevation}>
+                                            {networks[cryptoToNetworkSymbol(option.value)!].name}
```

Comment:
Please handle case when there is undefined returned by networkToCryptoSymbol. (do not use ! :))